### ISSUE TITLE
Ubuntu 22.04 (zfs-2.1.2): dracut-install: ERROR: installing '/usr/lib*/gcc/**/libgcc_s.so*'

### ISSUE THREAD

**Issue by sxc731** [openzfs/zfs#13398]
### System information
Type | Version/Name
 --- | ---
Distribution Name	| Ubuntu
Distribution Version	| 22.04
Kernel Version	| 5.15.0-27-generic
Architecture	| amd64
OpenZFS Version	| zfs-2.1.2-1ubuntu3 ;  zfs-kmod-2.1.2-1ubuntu3

### Describe the problem you're observing

During an Ubuntu 21.10 to 22.04 upgrade (using the standard `do-release-upgrade`) I noticed the following error:
```
dracut: *** Including module: zfs ***
dracut-install: ERROR: installing '/usr/lib*/gcc/**/libgcc_s.so*'
dracut: FAILED: /usr/lib/dracut/dracut-install -D /var/tmp/dracut.SRBFqW/initramfs -a /usr/lib*/gcc/**/libgcc_s.so*
```
### Describe how to reproduce the problem

On an Ubuntu 22.04 system that runs the v2.1.2 bundled version of ZFS and uses `dracut`, as opposed to the (more standard on Ubuntu) `initramfs-tools`, any attempt to run `dracut` reproduces the issue.

### Workaround
The issue appears to stem from a syntax error on [line 61 of 90zfs/module-setup.sh](https://github.com/openzfs/zfs/blob/af88d47f1ee86fae4abcfe0dabca67ea3c94d377/contrib/dracut/90zfs/module-setup.sh.in#L61); once corrected per the following workaround, the error goes away:
```
sed -e '/\*\*/s/\*\*/*\/*/' /usr/lib/dracut/modules.d/90zfs/module-setup.sh
```
This simply inserts a `/` between both `**`, and line 61 becomes:
```
dracut_install /usr/lib*/gcc/*/*/libgcc_s.so*
```
### More info
Thankfully I picked up the issue before rebooting; I assume that failure to do so might have prevented my system to boot properly but this is just speculation.

On Ubuntu 22.04, the file that this blob of code wants to deploy is located at: `/usr/lib/gcc/x86_64-linux-gnu/11/libgcc_s.so`, which explains why previous attempts to discover it fail.  Note that on a running system, `ldd /usr/sbin/zpool | grep -qF 'libgcc.*.so'` returns nothing so perhaps this lib is redundant here?

The syntax issue appears to have been corrected in master, as well as the version tagged as v2.1.99.  It is still present in the [2.1.5 staging branch](https://github.com/openzfs/zfs/blob/ce8d41ef75e1667710a5aeb393877c7ed4b34576/contrib/dracut/90zfs/module-setup.sh.in#L61).

It should be noted that this appears to be a [regression compared to v2.0.6](https://github.com/openzfs/zfs/blob/ef686e96ec1f907cac39552fa0e21298a6f38df5/contrib/dracut/90zfs/module-setup.sh.in#L58), which is what Ubuntu 21.10 was running (fat finger?)

Since a fix is already in hand, I'm just raising this issue in case someone else runs into this issue, since Ubuntu 22.04 LTS is bound to be a popular deployment.  It might be worth sweeping the fix into v2.1.5 so upcoming Ubuntu 22.04 point releases can pick it up.

**Comment by Dragon-debian** [comment]
>  Note that on a running system, ldd /usr/sbin/zpool | grep -qF 'libgcc.*.so' returns nothing so perhaps this lib is redundant here?

No its a workaround for #4749

**Comment by behlendorf** [comment]
I've opened #13487 against the 2.1.5-staging branch to apply the dracut fixes from the master branch.

**Comment by sxc731** [comment]
Confirmed as fixed in the latest Ubuntu 22.04 drop, version: zfs-2.1.5-1ubuntu6~22.04.1
